lib/route/route.js

```diff
@@ -304,16 +304,16 @@ const checkNextRepeatable = (_id, params, userData) => {
 //   if (!repeatable) {
```

Comment:
Remove all this commented out code?

Reply:
For another time - time is of the essence here